includes/admin/class-admin.php

```diff
@@ -1162,6 +1173,33 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'administrator' ) ) {
+				return;
+			}
+
+			if ( ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			$this->messages[] = array(
+				str_replace( array( '[', ']' ), array( '<a href="https://wordpress.org/plugins/font-awesome" target="_blank" rel="noopener nofollow">', '</a>' ), _x( 'We changed how FontAwesome is integrated in Business Directory. Go [here] to install the officle "FontAwesome styles" plugin.', 'admin', 'business-directory-plugin' ) ),
+				'error dismissible',
+				array( 'dismissible-id' => 'fontawesome_dismissed' ),
```

Comment:
This name is making the function name super long. Can we condense this down to just 'fa' or something?

Reply:
'FA' might not be a name common to most users

Reply:
I've changed it to `sprintf` and made the line shorter

Reply:
I'm specifically referring to "fontawesome_dismissed" since it's creating a hook like "wpbdp_admin_ajax_dismiss_notification_fontawesome_dismissed". Include "dismissed" is overkill and making for really long function names.

Reply:
Changes done :)

Reply:
Thanks! Looks like the tabs vs spaces are still messed up throughout all these changes.